gci/models.py

```diff
@@ -1 +1,27 @@
-# Create your models here.
+import json
+from django.db import models
+
+
+class Students(models.Model):
+    id = models.IntegerField(default=0, primary_key=True)
+    display_name = models.CharField(max_length=100, default='')
+    organization_id = models.IntegerField(default=0)
```

Comment:
Well, if we're moving to use of multiple orgs, it'd be better to have a separate model for orgs too.

Reply:
We only have `organization_id` and `organization_name` from the API, but there is a lot of organisation fields which org admins push into the system, like the logo & summary blurb, and like the common footer that appears on every task.

Also most of the information about other orgs is publicly accessible, so it is theoretically quite simple to create a static dataset of all tasks by all orgs.

I think it makes sense to create a very basic GoogleCodeInOrg , which students are linked to.